src/components/PropEditors.js

```diff
@@ -167,7 +167,7 @@ class NumberEditor extends React.Component {
   }
 
   componentWillReceiveProps(props) {
-    this.setState({ value: String(props.value) });
```

Comment:
anyway, if with this line


- you can edit a case that has a number field
- you can change the number
- you can save it
- editing it again shows you the new number

then i'm good!